x-pack/plugins/spaces/public/views/management/_manage_spaces.scss

```diff
@@ -1,9 +1,3 @@
-.spcManagePage__content {
-  max-width: 640px;
-  margin-left: auto;
-  margin-right: auto;
-  flex-grow: 0;
-}
-.spcGridPage {
+.spcGridPage, .spcManagePage {
   max-width: map-get($euiBreakpoints, 'xl');
```

Comment:
This is now an unecessary hack. You can just add the `restrictWidth` prop to these components and remove this max-width.